### DISCUSSION TITLE
RadioGroup legend has cursor: pointer

### DISCUSSION THREAD

**Discussion by synnottd** [twilio-labs/paste#1354] [Q&A] [answered]
Hi,

I was wondering was it intentional that the RadioGroup legend has `cursor: pointer`. I might be missing something but I think it is because you are using your `Label` component with `as="legend"`. https://github.com/twilio-labs/paste/blob/main/packages/paste-core/components/inline-control-group/src/InlineControlGroup.tsx#L29. 

I guess when you get down to it this is a little pedantic since it is based on what your interpretation of the pointer cursor is supposed to indicate. Just thought I'd ask since it seemed weird to me.

**Comment by corinne-nagel** [comment, marked as answer]
Hi @synnottd, we agree and created a ticket in our backlog to fix this. Thanks for letting us know! https://issues.corp.twilio.com/browse/DSYS-2740

**Reply by synnottd** [reply]
Perfect thanks!

**Reply by shleewhite** [reply]
Hi @synnottd - the fix for this has been merged: https://github.com/twilio-labs/paste/pull/2087. Thanks for letting us know about this bug!
